DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -37,131 +36,133 @@
+                  Welcome
+                  <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                    <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                  </svg>
+                </a>
+              </li>
+              <li class="nhsuk-header__navigation-item">
+                <a class="nhsuk-header__navigation-link" asp-controller="FindYourCentre" asp-action="Index">
+                  Find your centre
+                  <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                    <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                  </svg>
+                </a>
+              </li>
+              <li class="nhsuk-header__navigation-item">
+                <a class="nhsuk-header__navigation-link" asp-controller="Pricing" asp-action="Index">
+                  Pricing
+                  <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                    <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                  </svg>
+                </a>
+              </li>
+            }
+            @{
+              string helpPage;
+              if (User.HasClaim(c => c.Type == CustomClaimTypes.UserAdminId)) {
+                helpPage = "/help/Introduction.html";
+              } else if (User.Identity.IsAuthenticated) {
+                helpPage = "/learningportal/help/LearningPortal.html";
+              } else {
+                helpPage = "/help/Introduction.html";
               }
+            }
+            <li class="nhsuk-header__navigation-item">
+              <a class="nhsuk-header__navigation-link" href="@(Configuration["CurrentSystemBaseUrl"] + helpPage)" target="_blank">
+                Help
+                <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                  <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                </svg>
+              </a>
+            </li>
+            @if (!User.Identity.IsAuthenticated) {
               <li class="nhsuk-header__navigation-item">
-                <a class="nhsuk-header__navigation-link" href="@(Configuration["CurrentSystemBaseUrl"] + helpPage)" target="_blank">
-                  Help
+                <a class="nhsuk-header__navigation-link" asp-controller="Login" asp-action="Index">
+                  Log in

```

Comment:
The ticket doesn't seem to suggest that the log in/out/register links should be kept on the tracking system nav bar. It looks like they will be present (though the highlighted changes for _Layout are a bit jumbled so I might be missing something). Was it decided that these should still be present?

Reply:
It is only accessible when logged in, and HEE has made a decision to remove the Log out button for the Tracking system

Reply:
I think the only actual change here is around line 164, the rest is just page formatting changes from the auto formatter

Reply:
Good point. In which case I think this looks good.

Reply:
Though merge conflict might be a pain to solve.

Reply:
I'll let you know if I have trouble, I assume it is conflicting with some of your changes but I haven't checked yet